### PR TITLE
Fixes @less on windows (line off-by-one)

### LINKED ISSUE #27017: @less misses by one line on Windows
Linux:
```julia
julia> @less 2+2
(+)(x::T, y::T) where {T<:BitInteger} = add_int(x, y)
(*)(x::T, y::T) where {T<:BitInteger} = mul_int(x, y)

inv(x::Integer) = float(one(x)) / float(x)
(/)(x::T, y::T) where {T<:Integer} = float(x) / float(y)
[SNIP]
```
Windows 10:
```julia
julia> @less 2+2
(*)(x::T, y::T) where {T<:BitInteger} = mul_int(x, y)

inv(x::Integer) = float(one(x)) / float(x)
(/)(x::T, y::T) where {T<:Integer} = float(x) / float(y)
[SNIP]
```
Same result on 0.6.2 and latest 0.7 build. Missing by one line makes the macro completely useless on Windows since you can't page backwards with `more`.

### PR DESCRIPTION
**Pull request by rapus95**
Fixes https://github.com/JuliaLang/julia/issues/27017

### CODE REVIEW

**Comment by rapus95** [comment]
@StefanKarpinski when reviewing #30246 please merge this  code preview realted request too. (Should be fairly easy to review as it are only 6 basic lines of code 😄)

**Comment by rapus95** [comment]
@StefanKarpinski you touched that file recently, so I guess you are the right one to tag. It's 6 lines of fix but the issue is very annoying. Would be glad if it makes it into 1.14 aswell. Feature freeze shouldn't be relevant here as that is merely a bug fix on windows.

File: stdlib/InteractiveUtils/src/editless.jl [hunk 1]
```diff
@@ -108,7 +108,12 @@ edit(m::Module) = edit(pathof(m))
 if Sys.iswindows()
     function less(file::AbstractString, line::Integer)
         pager = shell_split(get(ENV, "PAGER", "more"))
-        g = pager[1] == "more" ? "" : "g"
+        if pager[1]=="more"
```
Comment (StefanKarpinski) [inline stdlib/InteractiveUtils/src/editless.jl]:
```suggestion
        if pager[1] == "more"
```

**Review by StefanKarpinski — APPROVED** [review]
I'm not sure how to evaluate this, but I guess it seems fine?

**Comment by rapus95** [comment]
On Windows it would just be trying @less with any function and checking that the first line is printed correctly. But besides that, that doesn't change anything. Especially nothing changes on the other platforms

**Comment by rapus95** [comment]
@StefanKarpinski Spacing is fixed, so this should be ready to go. Or, to rephrase into a massively ambiguous wording:
LGTM :D

**Comment by rapus95** [comment]
bump

**Pull request merged**